core/vm/interpreter.go

```diff
@@ -112,17 +112,21 @@ func (in *Interpreter) Run(snapshot int, contract *Contract, input []byte) (ret
 		op    OpCode        // current opcode
 		mem   = NewMemory() // bound memory
 		stack = newstack()  // local stack
+		stackCopy = newstack()
```

Comment:
I tried removing this line and changing L151 to `stackCopy := newstack()`, but I get:

core/vm/interpreter.go:129: undefined: stackCopy
core/vm/interpreter.go:197: undefined: stackCopy